### PR TITLE
"Sort list in ascending order" - Sort command

### PR DESCRIPTION
**Pull request by Bogi1001**

### CODE REVIEW

File: src/seedu/addressbook/commands/Sort.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" 
+            + "Displays all the entries in alphabetical order.\n\t";
+	public static final String MESSAGE_SUCCESS = "Sorted List: %1$s";
+	
+	
+	 
+	 @Override
+	  public CommandResult execute() {
+	        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+	        Collections.sort((List<ReadOnlyPerson>) allPersons);
```
Comment (pyokagan) [inline src/seedu/addressbook/commands/Sort.java:21]: `ReadOnlyPerson` needs to be [Comparable](https://docs.oracle.com/javase/8/docs/api/java/lang/Comparable.html) in order to be able to be sorted with `Collections.sort()`.

Doing a search on Google for `Java Collections.sort comparable` gives me these links:
- https://www.mkyong.com/java/java-object-sorting-example-comparable-and-comparator/
- https://docs.oracle.com/javase/tutorial/collections/interfaces/order.html

File: src/seedu/addressbook/commands/Sort.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+	    }
+
```
Comment (pyokagan) [inline src/seedu/addressbook/commands/Sort.java:24]: There is trailing whitespace on this line (and in many parts of this file). You can configure Eclipse to [strip trailing whitespace on save](http://stackoverflow.com/a/2618521)

File: src/seedu/addressbook/commands/Sort.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            + "Displays all the entries in alphabetical order.\n\t";
+	public static final String MESSAGE_SUCCESS = "Sorted List: %1$s";
+	
+	
+	 
+	 @Override
+	  public CommandResult execute() {
+	        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+	        Collections.sort((List<ReadOnlyPerson>) allPersons);
+	        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
```
Comment (pyokagan) [inline src/seedu/addressbook/commands/Sort.java:22]: Tabs mixed with spaces in the indentation of this line. You need to make sure you only use spaces for indentation.

**Review by pyokagan — CHANGES_REQUESTED** [review]
Fix these problems ASAP

**Comment by Bogi1001** [comment]
Changes made

**Comment by pyokagan** [comment]
What changes were made? I don't see any changes.

**Comment by Bogi1001** [comment]
Sent a new pull request with the edited code

**Pull request closed without merging**